helpers/index.js

```diff
@@ -11,6 +12,10 @@
    * var app = express();
    * app.use(helpers.errorHandler);
    * */
+
+  var gauge = new client.Gauge('nodejs_errors_total', 'This is a counter of all the exceptions handled.');
```

Comment:
Ok. I used a gauge because I was working on transforming it to a per/second base measurement. But I will remove it if.